src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1900,7 +1899,12 @@ def _oremote(self, meth, args, kwargs):
             raise NoOrchestrator()
 
         mgr.log.debug("_oremote {} -> {}.{}(*{}, **{})".format(mgr.module_name, o, meth, args, kwargs))
-        return mgr.remote(o, meth, *args, **kwargs)
+        try:
+            return mgr.remote(o, meth, *args, **kwargs)
+        except RuntimeError as e:
+            if meth != 'get_feature_set' and meth not in self.get_feature_set():
```

Comment:
why the special case for get_feature_set?  (maybe in a comment?)

Reply:
`self.get_feature_set` calls `self._oremote`, which calls `self.get_feature_set` here.